updateData.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
sorry, for file name I prefer snake case and without an extension so update_data

I also prefer no extension because of the "#!/usr/bin/env bash" we could change it without having the change the file (and "sh" isn't identical to bash :-))